app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb

```diff
@@ -34,17 +34,20 @@ def medications?
     end
 
     def add_intro
-      names = @patient.with_indifferent_access
-      full_name = [names[:first], names[:middle], names[:last]].reject(&:blank?).join ' '
-      patient_name = [full_name, names[:suffix]].reject(&:blank?).join ', '
+      patient_info = @patient.with_indifferent_access
+      full_name = [patient_info[:first], patient_info[:middle], patient_info[:last]].reject(&:blank?).join ' '
+      patient_name = [full_name, patient_info[:suffix]].reject(&:blank?).join ', '
+      birthdate = patient_info[:birthdate]
       generated_time = Time.now.getlocal
       generated_at = "#{generated_time.strftime('%m/%d/%Y')} at #{generated_time.strftime('%l:%M %p %Z')}"
 
       intro_lines = [
         "<font size='11'>Hypertension Rapid Ready for Decision | Claim for Increase</font>\n",
         "<font size='22'>VHA Hypertension Data Summary for</font>",
-        "<font size='22'>#{patient_name}</font>\n",
         "<font size='10'><i>Generated automatically on #{generated_at}</i></font>\n",
+        "<font size='11'>\n</font>",
+        "<font size='14'>#{patient_name}</font>\n",
+        "<font size='11'>DOB: #{birthdate}</font>\n",
```

Comment:
If you're unsure about the availability of `birthdate`, surround it in a conditional block.